Makefile

```diff
@@ -106,3 +106,7 @@ run: copy-es-certs
 		--cl-infra-role-name=sg_role_admin \
 		--ssl-insecure-skip-verify
 .PHONY: run
+
+run-linter:
```

Comment:
Suggest target name "lint:", same as CLO. Your call.
/lgtm
/hold

Reply:
updated to "lint"